docs/general_usage.md

```diff
@@ -115,6 +115,26 @@ ps -aux | twilio email:send \
   --text="See attachment"
 ```
 
+## Troubleshooting
+
+### Installing a specific version of twilio-cli
+
+General installation instructions are located [here](https://twil.io/cli#install-twilio-cli).
+
+A list of tagged releases can be found with `git ls-remote --tags https://github.com/twilio/twilio-cli.git`.
+
+Substitute X.X.X below with the desired version.
+
+#### Mac OS X
+
+`brew install https://raw.githubusercontent.com/twilio/homebrew-brew/X.X.X/Formula/twilio.rb`
```

Comment:
Need uninstall steps for brew.

Reply:
Those steps are given when you run that command by homebrew itself, if it's necessary.

Reply:
Ack.